examples/basic.py

```diff
@@ -0,0 +1,51 @@
+import argparse
+import json
+
+import grpc
+import matplotlib.pyplot as plt
+
+from armonik.client import TaskFieldFilter
+from armonik.stats import ArmoniKStatistics
+from armonik.stats.metrics import AvgThroughput, TotalElapsedTime, TimestampsTransition, TasksInStatusOverTime
+
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser("Compute statistics for tasks of a given session.")
+    parser.add_argument("--endpoint", "-e", type=str, help="ArmoniK controle plane endpoint")
+    parser.add_argument("--session-id", "-s", type=str, help="ID of the session")
+    args = parser.parse_args()
+
+    if args.endpoint.startswith("http://"):
```

Comment:
this line is not useful. You just need to use the last one. There are no error if the prefix is not there.